src/DynamoAnalyzer/AnalyzeEnvironment.cs

```diff
@@ -0,0 +1,18 @@
+using DynamoAnalyzer.Helper;
+
+namespace DynamoAnalyzer
+{
+    /// <summary>
+    /// Common dependencies used during the analysis
+    /// </summary>
+    internal static class AnalyzeEnvironment
+    {
+        private static DirectoryInfo _workspace;
```

Comment:
please don't use this naming pattern `_` - just use `workspace` etc.

Reply:
fixed